Puppet/Puppet.xcodeproj/project.pbxproj

```diff
@@ -792,6 +793,7 @@
 				INFOPLIST_FILE = Puppet/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
+				OTHER_LDFLAGS = "-all_load";
```

Comment:
Correct me if I'm wrong, but this means that everyone who wants to include our SDK needs to add this flag.
I'm not okay with this.